Libraries/Components/Slider/Slider.js

```diff
@@ -115,6 +116,24 @@ var Slider = React.createClass({
     thumbImage: Image.propTypes.source,
 
     /**
+     * Color of the foreground switch grip.
+     * @platform android
+     */
+    thumbTintColor: ColorPropType,
+
+    /**
+     * Color of the slider track.
+     * @platform android
+     */
+    trackTintColor: ColorPropType,
```

Comment:
Could we just use iOS props `minimumTrackTintColor` and `maximumTrackTintColor` instead of adding 2 new props?